osu.Game/Overlays/MusicController.cs

```diff
@@ -240,6 +240,7 @@ void preferUnicode_changed(object sender, EventArgs e)
 
         private void workingChanged(object sender = null, EventArgs e = null)
         {
+            progress.IsEnabled = (beatmapSource.Value != null);

```

Comment:
HasValue works only for nullable objects, actually using HasValue doesnt make difference unless if its for readability.